src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.spec.ts

```diff
@@ -528,7 +528,7 @@ describe('PoolFormComponent', () => {
       expect(component.getMaxSize()).toBe(3);
     });
 
-    it('should return the osd count as minimum if its lower the the rule minimum', () => {
+    it('should return the osd count as minimum if its lower the rule minimum', () => {
```

Comment:
check-spelling (the tool used to identify most of these items) can apply additional rules, e.g. the one here that complained about doubled words. (It could also be used to complain about contractions if someone were so inclined.)